<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_5">
    <widget class="QPushButton" name="guilb">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>251</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>범죄율 보기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="guilb3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>160</y>
       <width>251</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>광주광역시 현황</string>
     </property>
    </widget>
    <widget class="QLabel" name="backimg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>768</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="guilb2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>440</y>
       <width>251</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>결과보기</string>
     </property>
    </widget>
    <zorder>backimg</zorder>
    <zorder>guilb</zorder>
    <zorder>guilb3</zorder>
    <zorder>guilb2</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="lineEdit1">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>702</width>
       <height>321</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>경찰서</string>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>발생건수</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>검거건수</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>검거인원</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>구속</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>불구속</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>기타</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="back1">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>670</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>돌아가기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delb">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>210</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>삭제</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>530</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="backimg2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>768</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generation_btn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>610</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>관할별 범죄발생건수(%)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="arrest_unarrest_btn">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>610</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>관할별 범죄 검거/미검거건수(%)</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>500</y>
       <width>411</width>
       <height>24</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;   경찰서(str),발생건수,검거건수,검거인원,구속,불구속,기타 순으로 입력&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>backimg2</zorder>
    <zorder>generation_btn</zorder>
    <zorder>arrest_unarrest_btn</zorder>
    <zorder>lineEdit1</zorder>
    <zorder>tableWidget</zorder>
    <zorder>back1</zorder>
    <zorder>delb</zorder>
    <zorder>lineEdit2</zorder>
    <zorder>textBrowser_2</zorder>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QTextBrowser" name="textBrowser_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>811</width>
       <height>362</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Black'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;0.자료를 토대로 보면 검거 인원이 검거 횟수에 비해 10~20%정도만 높게 나오는데&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;이를 통해 대부분의 범죄는 다수보다는 극소수나 단독으로 범죄를 일으킨다는 것 을 알 수 있다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;1.cctv가 범죄율을 줄이는 유력한 이유로 추측했지만 1천명당 cctv가&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;가장 많은 동구가 인구대비 범죄율이 120으로 가장높았다,&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;그러나 검거율도 가장 높은 85%대로, cctv는 범죄율을 줄이기보단&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;범죄후의 검거에 도움을 더 많이 준다는 결과를 도출할 수 있었다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;2.면적과의 상관관계는 크지않다.압도적인 너비차를 보여주는 광산구는 13km당이라는 유의미한 너비 대비 발생을 보여주지만&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;동구와 북구는 세배에 달하는 크기차이가 나지만 그 범위 차이는 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;25km와 30km정도로 많지않다&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;3.단순발생수는 인구가 가장 밀접한 걸로 확인되었다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:14pt; font-weight:600;&quot;&gt;범죄발생수와 인구수가 북구&amp;gt;광산&amp;gt;서부&amp;gt;남부&amp;gt;동부로 동일하다.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Black'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;도출한 정보&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>550</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Black'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;결과&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>450</y>
       <width>811</width>
       <height>245</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Black'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;범죄의 단순 발생은 인구 수에 많은 영향을 받고&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;cctv나 지역크기등은 영향이 없지는 않으나 큰 영향을 주지는 않았습니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;따라서 범죄의 발생은 정리 가능한 단순한 몇가지로 인해 발생 하는것이 아닌,&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;당연하다고 생각하는 인구 수 같은 점부터 우발성, 시민의식, 주위 환경 등 보이지 않거나 수치화 할 수 없는 다른 요소들의 복합적인 요인으로 인해 발생한다는 것을 추측할수 있었습니다.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="backimg4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>1024</width>
       <height>768</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>backimg4</zorder>
    <zorder>textBrowser_3</zorder>
    <zorder>textBrowser_4</zorder>
    <zorder>textBrowser_5</zorder>
    <zorder>textBrowser_6</zorder>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QPushButton" name="gwangsan">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>530</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>광산</string>
     </property>
    </widget>
    <widget class="QPushButton" name="east">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>동구</string>
     </property>
    </widget>
    <widget class="QPushButton" name="west">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>서구</string>
     </property>
    </widget>
    <widget class="QPushButton" name="south">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>남구</string>
     </property>
    </widget>
    <widget class="QPushButton" name="north">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>450</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>북구</string>
     </property>
    </widget>
    <widget class="QPushButton" name="allspace">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>610</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>지역별평균</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nex">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>200</y>
       <width>161</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_2">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>310</y>
       <width>161</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>돌아가기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>511</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>         광주권 범죄율 그래프</string>
     </property>
    </widget>
    <widget class="PlotWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>471</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="allspace2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>610</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>지역별선형</string>
     </property>
    </widget>
    <widget class="QPushButton" name="allspace3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>610</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>지역별막대</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cl">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>610</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>초기화</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>460</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>410</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="backimg3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>768</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>170</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>발생인원</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>검거건수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>검거인원</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>구속</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>불구속</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>기타</string>
      </property>
     </item>
    </widget>
    <zorder>backimg3</zorder>
    <zorder>gwangsan</zorder>
    <zorder>east</zorder>
    <zorder>west</zorder>
    <zorder>south</zorder>
    <zorder>north</zorder>
    <zorder>allspace</zorder>
    <zorder>nex</zorder>
    <zorder>back_2</zorder>
    <zorder>label</zorder>
    <zorder>widget</zorder>
    <zorder>allspace2</zorder>
    <zorder>allspace3</zorder>
    <zorder>cl</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>comboBox</zorder>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
